src/components/landing/landingcsshelper.ts

```diff
@@ -0,0 +1,67 @@
+import { css } from 'styled-components'
+import { lighten } from 'polished'
+
+export const makeSVGStyle = ({ Parent }) => {
```

Comment:
all files in `components`-folder are expected to be components. CSS-Helper goes into helper, but in this case, the helper is tied to a component and shouldn't be exposed.